docker-images/kafka/exporter-scripts/kafka_exporter_run.sh

```diff
@@ -1,15 +1,18 @@
 #!/usr/bin/env bash
+set -e
 set +x
 
 # We don't need LOG_DIR because we write no log files, but setting it to a
 # directory avoids trying to create it (and logging a permission denied error)
 export LOG_DIR="$KAFKA_EXPORTER_HOME"
 
 if [ -n "$KAFKA_EXPORTER_GROUP_REGEX" ]; then
+    # shellcheck disable=SC2027
     groupregex="--group.filter=\""${KAFKA_EXPORTER_GROUP_REGEX}"\""
```

Comment:
```
groupregex="--group.filter=\"${KAFKA_EXPORTER_GROUP_REGEX}\""
```
Should mean we can remove the disable

Reply:
I'm not completely sure about it - I think removing it breaks the functionality with some settings.